backend/box/box.go

```diff
@@ -120,6 +120,7 @@ type Object struct {
 	size        int64     // size of the object
 	modTime     time.Time // modification time of the object
 	id          string    // ID of the object
+	publicLink  string    // Public Link for the object
```

Comment:
Is the publicLink mostly empty?  I'm just wondering how much this will bloat memory usage as sometimes rclone has millions of Object~s in memory.

Reply:
It should be empty for every object that isn't shared.

Reply:
I think if it is mostly empty then that is fine.